cmd/algorand-indexer/daemon.go

```diff
@@ -185,6 +197,8 @@ func init() {
 	daemonCmd.Flags().Uint32VarP(&maxApplicationsLimit, "max-applications-limit", "", 1000, "set the maximum allowed Limit parameter for querying applications")
 	daemonCmd.Flags().Uint32VarP(&defaultApplicationsLimit, "default-applications-limit", "", 100, "set the default Limit parameter for querying applications, if none is provided")
 
+	daemonCmd.Flags().StringVarP(&indexerDataDir, "datadir", "i", "", "path to indexer data dir, or $INDEXER_DATA")
```

Comment:
nit: kebab case used for other parameters
```suggestion
	daemonCmd.Flags().StringVarP(&indexerDataDir, "data-dir", "i", "", "path to indexer data dir, or $INDEXER_DATA")
```